testing-common/src/main/java/io/opentelemetry/javaagent/testing/common/AgentClassLoaderAccess.java

```diff
@@ -24,7 +24,7 @@
     }
   }
 
-  static Class<?> loadClass(String name) {
+  public static Class<?> loadClass(String name) {
```

Comment:
something like this, just so we don't accidentally make it package-private in the future?
```suggestion
  // public for use by downstream distros
  public static Class<?> loadClass(String name) {
```